docs/3.0.0-beta.x/guides/client.md

```diff
@@ -44,15 +44,16 @@ module.exports = strapi => {
 
 When the hook is created, we have to enable it to say to Strapi to use this hook.
 
-**Path —** `./config/hook.json`
-
-```json
-{
-  ...
-  "github": {
-    "enabled": true
-  }
-}
+**Path —** `./config/hook.js`
```

Comment:
again a note about creating this file (believe it or not users get really confused about this type of stuff especially with extensions)